Guides/Image_Formats.md

```diff
@@ -2,46 +2,44 @@
 
-The MDS format is, once again, similar to CUE and CCD in the basic structure. The .MDS file is similar to the .CCD file in that it again contains information about the disc's structure (along with additional data), while *not* being presented in a human readable format. The .MDF file is also similar to the .BIN and .IMG files, except that it can also contain the subchannel data directly, without using a .SUB file. This has pros and cons, with the main pro being that there are less files present, decluttering things slightly. The main con is that subchannel data is essentially never consistently dumped, meaning that an .MDF file that contains this data will not be usable for reliable DAT'ing, where the exact file hash needs to be consistent between dumps. Otherwise, the content of the .MDF file is equivalent to the data present in the .BIN and .IMG files where applicable.
+The main reason to use MDS for CDs is the fact that it supports more devious copy protection schemes directly, most notably ones that make use of DPM, in which the disc is mastered to intentionally make certain sectors take non-standard amounts of time to read, and these timings are measured, stored within the .MDS file, and then used to verify the disc's authenticity. Only certain very specific virtual drives support this feature, including Alcohol 120% (the program this format was originally designed for) and Daemon Tools.
 
-The main use-case of this format is the fact that it supports more devious copy protection schemes directly, most notably ones that make use of DPM. Essentially, some copy protections will use the time it takes to read certain sectors to verify the authenticity of a disc, where the disc has been mastered to purposefully make these timings non-standard. Alcohol 120%, the program this format was originally designed for, is able to make measurements of these timings and store this data within the .MDS file. This data is only usable with very specific virtual drives, including Alcohol 120% and Daemon Tools.
+MDS is also the simplest and most accessible format for copy-protected DVDs, as the .MDS file can contain additional data needed for most DVD copy protections, such as DMI. It's also worth noting that for DVDs, which don't have subchannels, the .MDF file should always match the .ISO file created by most other software. This means that if you have access to the necessary data, you can craft a functioning .MDS file yourself and rename a standard .ISO file to create a functioning image.
```

Comment:
It isn't necessary to point out that DVDs don't have subchannels here. The reason why an MDF can't simply be renamed to ISO for CDs is due to several factors other than just subchannels.